packages/medusa-plugin-wishlist/src/api/store/customers.js

```diff
@@ -1,107 +1,75 @@
-import { Router } from "express"
 import bodyParser from "body-parser"
-import { Validator, MedusaError } from "medusa-core-utils"
+import { Router } from "express"
 import jwt from "jsonwebtoken"
+import { MedusaError } from "medusa-core-utils"
 
 const JWT_SECRET = process.env.JWT_SECRET || ""
 
 export default () => {
   const app = Router()
 
   app.delete("/:id/wishlist", bodyParser.json(), async (req, res) => {
-    const schema = Validator.object().keys({
```

Comment:
todo: add wrapMiddleware to handle the handle any error through the error-handler middleware globally applied

Reply:
same for all end points